ServiceDesk/ServiceDeskClient.cs

```diff
@@ -0,0 +1,92 @@
+﻿namespace ServiceDesk
+{
+    using ServiceDesk.Abstractions;
+    using ServiceDesk.Abstractions.DTOs;
+    using System.Collections.Generic;
+    using System.Net.Http;
+    using System.Threading.Tasks;
+    using System.Text.Json;
+    using System.Threading;
+
+    public class ServiceDeskClient : IServiceDeskClient
+    {
+        private const string requestTypesUrl = "https://arcadia-servicedesk-dev31.azurewebsites.net/api/intra/requestTypes";
```

Comment:
Absolute API url should be in configuration. Client should only know about API endpoints urls.

Reply:
Fixed